src/tic_tac_toe/input.clj

```diff
@@ -5,13 +5,20 @@
 
 (defn prompt-terminal [question]
    (println question)
-   (read-string(read-line)))
+   (read-line))
 
 (defn valid-selection [input board]
   (and (number? input) (validmove? input) (moveopen? board input)))
 
-(defn user-input-move [board ]
- (let [input (prompt-terminal "what is your next move ?")]
+(defn user-marker []
```

Comment:
Also, if I input a move that is already taken the game crashes `Wrong number of args (2) passed to: input/user-input-move`